How-to-log-life.md

```diff
@@ -0,0 +1,115 @@
+# How to download a copy of everything Facebook knows about you
```

Comment:
I guess this file can be deleted now as there is a copy in the `Documentation` folder.